src/Storage/Storage.Management/help/Get-AzStorageLocalUser.md

```diff
@@ -0,0 +1,160 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/powershell/module/az.storage/get-azstoragelocaluser
+schema: 2.0.0
+---
+
+# Get-AzStorageLocalUser
+
+## SYNOPSIS
+Gets a specified local user or lists all local users in a storage account.
+
+## SYNTAX
+
+### AccountName (Default)
+```
+Get-AzStorageLocalUser [-ResourceGroupName] <String> [-StorageAccountName] <String> [-UserName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### AccountObject
+```
+Get-AzStorageLocalUser -StorageAccount <PSStorageAccount> [-UserName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzStorageLocalUser** cmdlet gets a specified local user or lists all local users in a storage account.
+
+## EXAMPLES
+
+### Example 1: Get a specified local user
+```
+PS C:\> $localUser = Get-AzStorageLocalUser -ResourceGroupName "myresourcegroup" -AccountName "mystorageaccount" -UserName testuser1
+
+PS C:\> $localUser 
+
+   ResourceGroupName: myresourcegroup, StorageAccountName: mystorageaccount
+
+Name      Sid                                          HomeDirectory HasSharedKey HasSshKey HasSshPassword PermissionScopes
+----      ---                                          ------------- ------------ --------- -------------- ----------------
+testuser1 S-1-2-0-0000000000-000000000-0000000000-0000 /             True         True      True           [container1,...]
+
+PS C:\> $localUser.PermissionScopes
+  
+Permissions Service ResourceName
+----------- ------- ------------
+rw          blob    container1  
+rw          file    share2
+```
```

Comment:
This example needs to be rewritten in the form of
````markdown
```powershell
$localUser = Get-AzStorageLocalUser ..... (note: "PS C:|>" needs to be deleted)
```

```output
```
````

If it cannot be written this way, please follow [this troubleshooting guide](https://github.com/Azure/azure-powershell/blob/main/documentation/Debugging-StaticAnalysis-Errors.md#example-add-skip-tag-to-the-example-whose-outputs-cannot-be-separated-from-codes:~:text=If%20outputs%20cannot%20be%20separated%20from%20codes%2C%20then%20please%20add%20the%20tag%20%3C!%2D%2D%20Skip%3A%20Output%20cannot%20be%20splitted%20from%20code%20%2D%2D%3E%20to%20the%20next%20line%20of%20the%20example%20title%20and%20in%20front%20of%20the%20code%20block.%20The%20following%20is%20an%20example.) to skip checking examples.
Thanks

Reply:
This example 1 has mutiple output , so add the skip tag.
For example 2, it has single output, so change per the format.

I have also updated other help to align with the format.